### PR TITLE
Update PHP-FPM and Varnish to use Centos8 Base Images

### PR DESCRIPTION
**Pull request by davidalger**
* Upstream PHP images are being updated to build atop a EL 8 vs EL 7: https://github.com/davidalger/docker-images-php/pull/6
* IUS repositories will no longer be present in upstream base images (as of this writing they were only used to install newer versions of Git and MariaDB which were not available in EL 7 repositories)
* Git 2.27 is installed from EL 8 repositories vs Git 2.22 (`git222` from IUS)
* MariaDB 10.3 client is installed from EL 8 repositories vs MariaDB 10.2 client (`mariadb102` from IUS)
* The newer curl library in EL 8 will now allow constants for newer curl library features to be used in PHP code (see https://github.com/davidalger/docker-images-php/issues/5 for details)

### CODE REVIEW

File: images/varnish/4.1/Dockerfile [hunk 1]
```diff
@@ -1,15 +1,15 @@
 FROM centos:7
```
Comment (tdgroot) [inline images/varnish/4.1/Dockerfile:1]: Base image is still centos:7

File: images/varnish/6.0/Dockerfile [hunk 2]
```diff
@@ -1,15 +1,15 @@
 FROM centos:7
```
Comment (tdgroot) [inline images/varnish/6.0/Dockerfile:1]: Base image is still centos:7

File: images/varnish/6.4/Dockerfile [hunk 3]
```diff
@@ -1,15 +1,15 @@
 FROM centos:7
```
Comment (tdgroot) [inline images/varnish/6.4/Dockerfile:1]: Base image is still centos:7

File: images/varnish/6.5/Dockerfile [hunk 4]
```diff
@@ -1,15 +1,15 @@
 FROM centos:7
```
Comment (tdgroot) [inline images/varnish/6.5/Dockerfile:1]: Base image is still centos:7

**Comment by davidalger** [comment]
Testing these has involved building the images from the upstream PR to provide EL8 base php images and then:

```
./images/scripts/build.sh php-fpm
./images/scripts/build.sh php-fpm/magento2
./images/scripts/build.sh varnish
```

Once the images are built locally, used a [copy of this template](https://github.com/davidalger/warden-env-magento2) to launch a Magneto 2 environment (without pulling during bootstrap to ensure locally built images are used):

```
warden bootstrap --clean-install --no-pull
```

After this completes, verify the application is working and operational in a browser, and verify `/etc/os-release` correctly report use of CentOS 8 vs CentOS 7.

**Comment by tdgroot** [comment]
LGTM, have been using this configuration for several weeks now!

**Comment by davidalger** [comment]
Now that the upstream php-fpm images are rebuilt on CentOS 8, I did a few things today to test this as well and everything appears to be A-ok.

* Rebuilt the images here on EL 8 base images: https://github.com/davidalger/docker-images-magento
* Launched an M2 Demo instance using these pre-built init images leveraging the project here: https://github.com/davidalger/m2demo/ via `warden bootstrap --no-pull`
* Created a from-scratch Magento 2.4 installation using my project template (https://github.com/davidalger/warden-env-magento2) and `warden bootstrap --clean-install --no-pull`
* Created a from-scratch Shopware 6 environment and installed with sample data successfully.

In each case I double checked I was working with EL 8 images by running `warden shell` and then `cat /etc/os-release`

Going to get this merged as soon as I can write up a change log entry and upgrade notes regarding this given it is a rather larger change.

**Pull request merged**